wave_lang/kernel/wave/utils/run_utils.py

```diff
@@ -6,182 +6,35 @@
 import functools
 import os
 from itertools import chain
-from typing import Any, Callable, Optional
-from warnings import warn
+from typing import Any, Callable
 
-import iree.runtime as rt
 import torch
 
 from wave_lang.kernel.lang import IndexSymbol
 
 from ..compile_options import WaveCompileOptions
-from ..profiling import benchmark_module
-
-# Cache for the system context and vm function.
-RUNTIME_CACHE: dict[str, tuple[rt.SystemContext, rt.VmFunction]] = {}
 
```

Comment:
Could you briefly explain the effect this and `invoke_vmfb` has on caching? Do we not cash at all when using wave runtime and only using iree runtime or not entirely anymore? Is this documented somewhere?

Reply:
`Launchable` is doing the same type of caching internally.